plugins/kubernetes-backend/src/plugin.ts

```diff
@@ -21,15 +21,46 @@ import {
 } from '@backstage/backend-plugin-api';
 import { catalogServiceRef } from '@backstage/plugin-catalog-node/alpha';
 
-import { KubernetesBuilder } from '@backstage/plugin-kubernetes-backend';
+import {
+  KubernetesBuilder,
+  KubernetesObjectsProvider,
+} from '@backstage/plugin-kubernetes-backend';
+import {
+  KubernetesObjectsProviderExtensionPoint,
+  kubernetesObjectsProviderExtensionPoint,
+} from '@backstage/plugin-kubernetes-node';
 
 /**
  * This is the backend plugin that provides the Kubernetes integration.
  * @alpha
  */
+
+class ObjectsProvider implements KubernetesObjectsProviderExtensionPoint {
+  private clusterSuppliers: KubernetesObjectsProvider | undefined;
```

Comment:
copy-paste error I think
```suggestion
  private objectsProvider: KubernetesObjectsProvider | undefined;
```

Reply:
Yep thank you !